internal/fuzzbinder/fuzzscraper.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package fuzzbinder
+
+// This file provides the scraper module that helps to get the fuzzing
+// statistics from the ClusterFuzz and OSS-Fuzz reports.
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+)
+
+// CoverageBucket is the OSS-Fuzz bucket containing the coverage reports.
+const CoverageBucket = "oss-fuzz-coverage"
+
+// CoverageSummary contains the coverage report summary.
+type CoverageSummary struct {
+	Data    []SummaryData `json:"data"`
+	Type    interface{}   `json:"type"`
+	Version interface{}   `json:"version"`
+}
+
+// SummaryData contains the data of the coverage report summary.
+type SummaryData struct {
+	Files  interface{}                     `json:"files"`
+	Totals map[string](map[string]float64) `json:"totals"`
+}
+
+// GetBucket gets a GCS bucket given its name.
+func GetBucket(bucketName string) (*storage.BucketHandle, error) {
```

Comment:
Why are `GetBucket`, `ListBlobs` and `GetBlob` public? Do you expect them to be called by functions outside this package?

Reply:
Thanks for the comment :)) I have changed them. I will make them public if I need to call them outside the package. For now, I'm not sure if I'll need them.